commitlint.config.js

```diff
@@ -1,15 +1,14 @@
-const { types, scopes } = require('./commitizen.config');
+const { types, scopes } = require("./commitizen.config");
 
-const validTypes = types.map((type) => type.value);
-const validScopes = scopes.map((scope) => scope.name);
+const validTypes = types.map(type => type.value);
+const validScopes = scopes.map(scope => scope.name);
 
 module.exports = {
-  extends: ['@commitlint/config-angular'],
+  extends: ["@commitlint/config-angular"],
 
   rules: {
-    'body-leading-blank': [2, 'always'],
-    'body-tense': [0, 'always', ['present-imperative']],
```

Comment:
Why was this removed?

Reply:
It has been removed in the linter upstream. Building with a fresh install via npm wouldn't work with it for me.